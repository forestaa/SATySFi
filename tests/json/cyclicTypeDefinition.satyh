type a = b
and  b = a
