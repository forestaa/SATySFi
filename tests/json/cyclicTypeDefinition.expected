{
  "file": "cyclicTypeDefinition.satyh",
  "range": {
    "start": { "line": 1, "character": 5 },
    "end": { "line": 2, "character": 6 }
  },
  "message":
    "! [Type Error] cyclic synonym type definition:\n      b (at \"cyclicTypeDefinition.satyh\", line 2, characters 5-6)\n      a (at \"cyclicTypeDefinition.satyh\", line 1, characters 5-6)"
}
